github/event_types.go

```diff
@@ -187,25 +187,23 @@ type TeamChange struct {
 	} `json:"repository,omitempty"`
 }
 
-// IntegrationInstallationEvent is triggered when an integration is created or deleted.
-// The Webhook event name is "integration_installation".
+// InstallationEvent is triggered when a GitHub App has been installed or uninstalled.
+// The Webhook event name is "installation".
 //
-// GitHub API docs: https://developer.github.com/early-access/integrations/webhooks/#integrationinstallationevent
-type IntegrationInstallationEvent struct {
-	// The action that was performed. Possible values for an "integration_installation"
-	// event are: "created", "deleted".
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#installationevent
+type InstallationEvent struct {
+	// The action that was performed. Can be either "created" or "deleted".
```

Comment:
@shurcooL do you want me to remove these comments? They duplicate the GitHub API and it's only a comment for a single field, yet the GitHub API have docs for other fields. So either these should be removed or all of the fields should have the docs duplicated IMHO - is there another reason they are there?